src/js/services/templating.js

```diff
@@ -0,0 +1,40 @@
+import Mustache from 'mustache';
+
+export function render(templateId, data) {
+  Mustache.tags = ['[[', ']]'];
+  const templateElement = document.getElementById(templateId);
+
+  if (templateElement) {
+    const output = Mustache.render(templateElement.innerHTML, data);
+    templateElement.insertAdjacentHTML('afterend', output);
+    templateElement.remove();
+  }
+}
+
+export const filters = {
+  ellipsis: (maxLength) => {
+    return  () => (text, renderer) => ellipsis(renderer(text), maxLength)
+  },
+  clean: (cleanRegExp, replacement) => {
+    return  () => (text, renderer) => renderer(text).replace(cleanRegExp, replacement)
+  },
+}
+
+function ellipsis(text, maxLenght) {
```

Comment:
I guess this is used instead of a normal css overflow ellipsis to make clean cuts on word breaks, right?

Reply:
yes; and it was brought from old landing :dagger: